core/include/kernel/interrupt.h

```diff
@@ -299,6 +300,16 @@ static inline void interrupt_raise_sgi(struct itr_chip *chip, size_t itr_num,
 	chip->ops->raise_sgi(chip, itr_num, cpu_mask);
 }
 
+/*
+ * interrupt_reset() - Reset the property for an interrupt
+ * @chip	Interrupt controller
+ * @itr_num	Interrupt number to reset
```

Comment:
This is supported only for SPIs, could you add this information in this inline description comment?

Reply:
Done